blocksuite/framework/block-std/src/gfx/viewport/turbo-renderer/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './turbo-renderer.js';
+export * from './types.js';
```

Comment:
Unacceptable block specific concept in types.ts

```ts
export interface ParagraphLayout {
  sentences: SentenceLayout[];
  zoom: number;
}
```

Reply:
This won't be away very soon, I'll find a workaround bridge the `TurboRenderer` with `GfxViewportElement`.

Reply:
You'll need to add extension load point in `TurboRenderer`, and let paragraph block register add it's only renderer.